tests/ui_tests/test_admin_page.py

```diff
@@ -20,140 +22,183 @@
-        users_page.open_settings_menu()
-        settings_page = AdminSettingsPage(users_page.driver, users_page.base_url)
-        settings_page.wait_page_is_opened()
-    with allure.step('Check Users menu'):
-        settings_page.open_users_menu()
-        users_page.wait_page_is_opened()
-    with allure.step('Check Intro menu'):
-        users_page.open_intro_menu()
-        intro_page = AdminIntroPage(users_page.driver, users_page.base_url)
+class TestAdminIntroPage:
+    """Tests for the /admin/intro"""
+
+    def test_open_by_tab_admin_intro_page(self, app_fs):
+        """Test open /admin/intro from left menu"""
+
+        users_page = AdminUsersPage(app_fs.driver, app_fs.adcm.url).open()
+        intro_page = users_page.open_settings_menu()
         intro_page.wait_page_is_opened()
+        intro_page.check_all_elements()
+        intro_page.check_admin_toolbar()
+
 
+class TestAdminSettingsPage:
+    """Tests for the /admin/roles"""
 
-def test_settings_filter(settings_page: AdminSettingsPage):
-    """Apply different filters on Admin Settings page"""
-    params = {
-        'search_text': 'ADCM',
-        'field_display_name': "ADCM's URL",
-        'group': 'Global Options',
-    }
-    get_rows_func = settings_page.config.get_all_config_rows
-    with allure.step(
-        f'Search {params["search_text"]} and check {params["field_display_name"]} ' 'is presented after search'
-    ):
-        with expect_rows_amount_change(get_rows_func):
-            settings_page.config.search(params['search_text'])
-        settings_page.config.get_config_row(params["field_display_name"])
-    with allure.step('Clear search'), expect_rows_amount_change(get_rows_func):
-        settings_page.config.clear_search()
-    with allure.step(
-        f'Click on {params["group"]} group and check {params["field_display_name"]} '
-        'is not presented after group roll up'
-    ):
-        with expect_rows_amount_change(get_rows_func):
-            settings_page.config.click_on_group(params['group'])
-        with pytest.raises(AssertionError):
+    def test_open_by_tab_admin_settings_page(self, app_fs):
+        """Test open /admin/settings from left menu"""
+
+        intro_page = AdminIntroPage(app_fs.driver, app_fs.adcm.url).open()
+        settings_page = intro_page.open_settings_menu()
+        settings_page.wait_page_is_opened()
+        settings_page.check_all_elements()
+        settings_page.check_admin_toolbar()
+
+    def test_settings_filter(self, settings_page: AdminSettingsPage):
+        """Apply different filters on Admin Settings page"""
+        params = {
+            'search_text': 'ADCM',
+            'field_display_name': "ADCM's URL",
+            'group': 'Global Options',
+        }
+        get_rows_func = settings_page.config.get_all_config_rows
+        with allure.step(
+            f'Search {params["search_text"]} and check {params["field_display_name"]} ' 'is presented after search'
```

Comment:
it can be one line, I guess, linter probably will find it

Reply:
it is the way black fix it